src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -212,7 +212,7 @@ private void SetupAuthorities(string aadAuthority, ProtocolVersion version)
         /// <exception cref="ArgumentNullException"> if <paramref name="securityToken"/> is null.</exception>
         /// <exception cref="ArgumentNullException"> if <paramref name="validationParameters"/> is null.</exception>
         /// <exception cref="SecurityTokenInvalidIssuerException">if the issuer is invalid or if there is a network issue. </exception>
-        internal async ValueTask<string> ValidateAsync(
+        public async ValueTask<string> ValidateAsync(
```

Comment:
there are github action warnings, can you please take a look at those.

Reply:
About to send an iteration it seems it doesnt recognize the null checks we do at the beginning